Bourreau/Gemfile

```diff
@@ -29,7 +29,7 @@ gem "net-ssh",  :require => 'net/ssh'
 gem "net-sftp", :require => 'net/sftp'
 gem "log4r"
 gem "mysql2"
```

Comment:
Should also restrict mysql to be older than version 0.4:

gem "mysql2", "<0.4"

Otherwise:

/home/glatard/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.5/lib/bundler/rubygems_integration.rb:271:in `block in replace_gem': Please install the mysql2 adapter: `gem install activerecord-mysql2-adapter` (can't activate mysql2 (~> 0.3.10), already activated mysql2-0.4.2. Make sure all dependencies are added to Gemfile.) (LoadError)
	from /home/glatard/.rvm/gems/ruby-2.2.0/bundler/gems/rails-d94d77c55c59/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb:3:in `<top (required)>'
	from /home/glatard/.rvm/gems/ruby-2.2.0/bundler/gems/rails-d94d77c55c59/activesupport/lib/active_support/dependencies.rb:251:in `require'
	from /home/glatard/.rvm/gems/ruby-2.2.0/bundler/gems/rails-d94d77c55c59/activesupport/lib/active_support/dependencies.rb:251:in `block in require'
	from /home/glatard/.rvm/gems/ruby-2.2.0/bundler/gems/rails-d94d77c55c59/activesupport/lib/active_support/dependencies.rb:236:in `load_dependency'
...